Tests/RenderingTests/Modifiers/TextInputAndOutputModifiersTests.swift

```diff
@@ -179,4 +179,49 @@ final class TextInputAndOutputModifiersTests: XCTestCase {
         }
         #endif
     }
+    
+    func testKeyboardShortcut() throws {
```

Comment:
Keyboard shortcuts are not rendered, so this test doesn't test anything besides modifier decoding. It should be removed.